app/assets/javascripts/client/inventories/InventoryPermission.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').factory('InventoryPermission', ['$resource', 'UrlService', function($resource, UrlService) {
```

Comment:
Ideally, this factory should follow the same pattern as all of our other code - separate out the definition of the factory into its own function and inject what we need to analogous to how we'd create a controller.